demisto_sdk/tests/integration_tests/run_integration_test.py

```diff
@@ -115,28 +115,26 @@ def test_json_to_outputs_flag_fail_no_prefix(
 ):
     """
     Given
-    - kl-get-components command
+    - kl-get-records command
 
     When
-    - Running `run` command on it with json-to-outputs flag and no prefix argument
+    - Running `run` command on runner instnce with json_to_outputs=True and
+      incident_id=pg_id
 
     Then
     - Ensure the json_to_outputs command is failing due to no prefix argument provided.
     """
     logger_info = mocker.patch.object(logging.getLogger("demisto-sdk"), "info")
     monkeypatch.setenv("COLUMNS", "1000")
     # mocks to allow the command to run locally
-    mocker.patch.object(Runner, "_get_playground_id", return_value="pg_id")
-    mocker.patch.object(Runner, "_run_query", return_value=["123"])
+    runner = Runner(query="kl-get-records", json_to_outputs=True, incident_id="pg_id")
+    mocker.patch.object(runner, "_run_query", return_value=["123"])
     # mock to get test log file
```

Comment:
please add a test (using mocker) that the incident_id value is passed as expected